mzLib/Test/Transcriptomics/TestData/HomozygousHLA.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<mzLibProteinDb>
+  <entry>
+    <accession>ENST00000395363</accession>
+    <name>ENST00000395363</name>
+    <gene>
+      <name type="primary">HLA-DQA1</name>
+      <name type="accession">ENSG00000196735</name>
+    </gene>
+    <organism>
+      <name type="scientific">Homo sapiens</name>
+    </organism>
+    <feature type="sequence variant" description="6\t32637480\t.\tC\tA\t13539.77\t.\tANN=A|missense_variant|MODERATE|HLA-DQA1|ENSG00000196735|transcript|ENST00000395363.5|protein_coding|1/5|c.22C&gt;A|p.Leu8Met|80/1206|22/768|8/255||\tGT:AD:DP:GQ:PL\t1/1:0,337:337:99:13568,1011,0">
```

Comment:
is "id" field accessible? what about the readability of alternative "description"? what about the "evidence" field?
![image](https://github.com/user-attachments/assets/e3ea9619-f189-4c62-a2e4-389b39f17a05)

Reply:
Readable in same fashion as Anthonys parser for protein variants